weave

```diff
@@ -1972,20 +1981,10 @@ EOF
             shift
         done
         [ -n "$SUB_STATUS" ] || echo
-        if status=$(call_weave GET $STATUS_URL) ; then
-            case "$status" in
-                "")
-                    ;;
-                "404 page not found")
-                    echo "Invalid 'weave status' sub-command: $SUB_COMMAND" >&2
-                    usage
-                    ;;
-                *)
-                    echo "$status"
-                    ;;
-            esac
-        else
-            res=1
+        call_weave GET $STATUS_URL || res=$?
+        if [ $res = 404 ] ; then
```

Comment:
I don't think return values can be greater than 255. The [POSIX spec](http://pubs.opengroup.org/onlinepubs/009695399/utilities/return.html) says:
> If the value of n is greater than 255, the results are undefined.